src/components/SalesChannel.js

```diff
@@ -0,0 +1,321 @@
+                       <div className="nfd-flex-1">
+                            <h2 className="nfd-text-[#0F172A] nfd-text-lg nfd-font-medium nfd-pb-1.5">
+                                    {__("Sell your products everywhere, confidently, with Ecomdash.", "wp-module-ecommerce")}
+                            </h2>  
+                            <p className="nfd-text-[#4A5567] nfd-pb-7 nfd-leading-5  nfd-font-medium">
+                                {__("With ecomdash, you can manage different sales channels all in one place.", "wp-module-ecommerce")}
+                                <br />
+                                {__("Simply connect your business accounts from Amazon, Etsy or Walmart to", "wp-module-ecommerce")}
+                                <br />
+                                {__("your ecomdash platform and easily manage products, customer orders and", "wp-module-ecommerce")}
+                                <br />
+                                {__("more saving you valuable time.", "wp-module-ecommerce")}​
+                            </p>
+                            <Button
+                                    className="nfd-button nfd-button--primary"
+                                    variant="primary" 
+                                    type="button"                                    
+                                    as={ canAccessGlobalCTB ?  hasEcomdash ? ecomdashStatus.data.isInstalled ? "a" : "button" : "button" : null }
+                                    data-ctb-id={
+                                    canAccessGlobalCTB && hasYithExtended && !hasEcomdash
+                                        ? "3edcf593-dbbe-4994-b5c0-a6718bb819c4"
+                                        : null
+                                    }
+                                    href= { (ecomdashStatus.data.isInstalled && ecomdashSetupStatus && ecomdashStatus?.data?.pluginUrl) || (ecomdashStatus.data.isInstalled && !ecomdashSetupStatus && "https://app.svc.ecomdash.com/ecom/dashboard") }
+                                    target="_blank"
+                                    isLoading={ecomdashStatus.data?.isInstalling}
+                                    onClick={hasEcomdash && ecomdashStatus.data.isNeedToInstall ? createPluginInstallAction(
+                                        'nfd_slug_ecomdash_wordpress_plugin',
+                                        16,
+                                        wpModules
+                                    ) : null}
+                                    id={ecomdashStatus.data.isInstalled ? "manage-ecomdash" : "install-ecomdash" }                                    
+                                    >
+                                    {ecomdashStatus.data.isInstalled ? ecomdashSetupStatus ? __("Get Started Now", "wp-module-ecommerce") : __("Go to Ecomdash", "wp-module-ecommerce") : __("Get Started Now", "wp-module-ecommerce") }
```

Comment:
```suggestion
                                    {ecomdashStatus.data.isInstalled ?  ( ecomdashSetupStatus ? __("Get Started Now", "wp-module-ecommerce") : __("Go to Ecomdash", "wp-module-ecommerce") ) : __("Get Started Now", "wp-module-ecommerce") }
```

Reply:
@ramyakrishnai As discussed, button label when in `to be installed` state will be `Install now`